internal/command/jsonplan/condition.go

```diff
@@ -0,0 +1,26 @@
+package jsonplan
+
+// conditionResult is the representation of an evaluated condition block.
+type conditionResult struct {
```

Comment:
Maybe consumers would benefit from having source location information for the condition too here? That could help a UI reporting condition status to link to the source code of the check block, so a user can see how the condition was defined.

Reply:
Possibly! But from collaborator discussions about how we might use this output, that doesn't seem a likely usage, so I'd rather not add it until there's demand.